packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -225,6 +254,42 @@ describe('components/Image', () => {
       expect(onLoadStub.mock.calls.length).toBe(1);
       expect(onLoadEndStub.mock.calls.length).toBe(1);
     });
+
+    test('is not called on update if "headers" and "uri" the same', () => {
```

Comment:
Not too sure what this test is going to assert. Should it say:

>if "headers" and "uri" have the same value

?

It just looks like we are rendering the same thing twice (or I'm having trouble seeing the change between the first and second render)

Reply:
A better name would be 
`is not called on update if "headers" and "uri" were not modified` (e.g. they didn't change)

Yes we're rendering the same thing twice, but since it's an inline declaration of an object, it 
is considered a different object (`===` would return `false`) 
It's a very common case and even in our code base we'd use the image like this 

```jsx
<Image source={{ uri: props.uri, headers: props.headers }} />
``` 

When the parent component re-renders, `source` would be considered a new object, but the value didn't really change, so we don't want to start loading the image again - we already did load it

I'll capture this as a comment remark above the test